ocean_lib/data_provider/data_service_provider.py

```diff
@@ -234,60 +236,45 @@ def sign_message(wallet: Wallet, msg: str) -> Tuple[str, str]:
         return nonce, sign_hash(encode_defunct(text=f"{msg}{nonce}"), wallet)
 
     @staticmethod
-    @enforce_types
+    # TODO reinstate @enforce_types
+    # @enforce_types
```

Comment:
I think you can already handle this TODO. Or do you want to leave it for the second step?

Reply:
I tried another quick attempt to get this working but was unsuccessful.  I'll take another look at it later.